tests-behat/bootstrap/Context.php

```diff
@@ -67,7 +67,7 @@ protected function disableAnimations(): void
             'animation-duration' => '5s',
             'transition-duration' => '5s',
         ]);
-        $script = '$(\'<style>' . $css . '</style>\').appendTo(\'head\'); $.fx.off = true;';
+        $script = 'if (Array.prototype.filter.call(document.getElementsByTagName("Style"), e => e.getAttribute("about") === "atk-test-behat").length === 0) {$(\'<style about="atk-test-behat">' . $css . '</style>\').appendTo(\'head\'); $.fx.off = true;}';
```

Comment:
https://stackoverflow.com/questions/14017864/how-to-check-if-a-tag-exists-using-javascript-without-getting-an-error

`if () {}` seems much simpler, also `getElementsByTagName("style")` (style, not Style)

Reply:
was this the issue or is it just randomly fixed?

running https://github.com/atk4/ui/tree/test_behat_fix more tests...

Reply:
yeah, need more testing. 

getElementsByTagName("style").length === 0 will always be false. What we need to check if there is a style tag using atk-test-behat attribute. I will fix typo. 

does https://github.com/atk4/ui/tree/test_behat_fix has the latest changes?

Reply:
yes, it has - see git tree, feel free to use this branch for testing, ie. merge this branch there, push, and recheck the CI results for that branch